doc/mgr/dashboard.rst

```diff
@@ -139,56 +139,50 @@ Status
 * **Cluster Status**: Displays overall cluster health. In case of any error it
   displays a short description of the error and provides a link to the logs.
 * **Hosts**: Displays the total number of hosts associated to the cluster and
-  links to the subpage displaying the list and descriptions of all hosts.
-* **Monitors**: Displays the total number of all MONs and their quorum status,
-  open sessions and links to the subpage providing a list and descriptions of all
-  the MONs.
-* **OSDs**: Displays total number of object storage daemons for Ceph (ceph-osds)
-  and the total number of OSDs running (up), total number of OSDs in the cluster
-  (in) and total number of OSDs out of the cluster (out). Provides link to the
-  subpage containing a list of all OSDs and related management actions.
-* **Managers**: Displays the total number of active and standby Ceph Manager
-  daemons (ceph-mgr) running alongside monitor daemons.
-* **Object Gateway**: Displays the total number of active object gateways and
-  provides a link to the subpage displaying a list of all object gateway daemons.
-* **Metadata Servers**: Displays total number of active and standby metadata
-  servers daemons for CephFS (ceph-mds).
-* **iSCSI Gateways**: Display the total number of iSCSI gateways available, total
-  number of active iSCSI gateways (up) and total number of inactive iSCSI
-  Gateways (down). Provides link to the subpage providing a list of all iSCSI
-  Gateways.
+  links to a subpage that lists and describes each.
+* **Monitors**: Displays mons and their quorum status and
+  open sessions and links to a subpage that lists and describes each.
+* **OSDs**: Displays object storage daemons (ceph-osds) and
+  the numbers of OSDs running (up), in service
+  (in), and out of the cluster (out). Provides links to
+  subpages providing a list of all OSDs and related management actions.
+* **Managers**: Displays active and standby Ceph Manager
+  daemons (ceph-mgr).
+* **Object Gateway**: Displays active object gateways (RGWs) and
+  provides links to subpages displaying list of all object gateway daemons.
```

Comment:
...provides links to subpages that display...

Reply:
ack